src/SprykerEco/Yves/Payone/Theme/default/klarna.twig

```diff
@@ -0,0 +1,6 @@
+<div class="payment-subform payone-form payone-invoice-form payment-form" id="payoneInvoice">
+    <div class="field">
+        {{ form_widget(form.payoneKlarna.pay_methods) }}
+        {{ form_widget(form.payoneKlarna.payMethodToken) }}
+    </div>
+</div>
```

Comment:
What is it?
We should follow the atomic structure: components(atom/molecule/organism), view, template.

Reply:
Removed as unused.